sretoolbox/container/image.py

```diff
@@ -331,17 +349,29 @@ def _raise_for_status(self, response, error_msg=None):
         _LOG.debug('[%s, %s]', str(self), msg)
         raise HTTPError(msg)
 
-    @retry(exceptions=(HTTPError, requests.ConnectionError), max_attempts=5)
-    def _request_get(self, url):
-        # Try first without 'Authorization' header
-        headers = {
-            'Accept':
-                'application/vnd.docker.distribution.manifest.v1+json,'
-                'application/vnd.docker.distribution.manifest.v2+json,'
-                'application/vnd.docker.distribution.manifest.v1+prettyjws,'
-        }
+    @classmethod
+    def _should_cache(cls, response):
+        """Returns whether to cache the response.
 
-        response = requests.get(url, headers=headers, auth=self.auth)
+        We cache responses with an etag that are not too large, to
+        protect the system from OOM
+
+        """
+        try:
+            return (response.ok and 'etag' in response.headers and
+                    int(response.headers['content-length']) <
+                    cls.MAX_CACHE_ITEM_SIZE)
+        except (KeyError, ValueError):
+            return False
+
+    @retry(exceptions=(HTTPError, requests.ConnectionError), max_attempts=1)
```

Comment:
Why re we reducing from 5 to 1 max_attempts?

Reply:
Mistake from another test, sorry about this one.